### PR TITLE
Fix type assertions

### PR DESCRIPTION
**Pull request by taktv6**

### CODE REVIEW

File: protocols/bgp/packet/path_attributes.go [hunk 1]
```diff
@@ -637,7 +637,7 @@ func (pa *PathAttribute) serializeCommunities(buf *bytes.Buffer) uint16 {
 		return 0
 	}
 
-	coms := pa.Value.(*[]uint32)
+	coms := pa.Value.(*types.Communities)
```
Comment (hikhvar) [inline protocols/bgp/packet/path_attributes.go]: I think we should at regression tests to ensure the communities are serialized correctly.

Comment (taktv6) [reply protocols/bgp/packet/path_attributes.go]: Absolutely. But not in this quick fix.

File: protocols/bgp/server/fsm_open_sent.go [hunk 2]
```diff
@@ -137,6 +137,7 @@ func (s *openSentState) openMsgReceived(openMsg *packet.BGPOpen) (state, string)
 
 func (s *openSentState) handleOpenMessage(openMsg *packet.BGPOpen) (state, string) {
 	s.fsm.holdTime = time.Duration(math.Min(float64(s.fsm.peer.holdTime), float64(time.Duration(openMsg.HoldTime)*time.Second)))
+	fmt.Printf("s.fsm.holdTime: %d\n", s.fsm.holdTime)
```
Comment (czerwonk) [inline protocols/bgp/server/fsm_open_sent.go]: in my opinion, this should be logging, but can be fixed later

Comment (taktv6) [reply protocols/bgp/server/fsm_open_sent.go]: Fixed it.

**Pull request merged**